packages/admin/admin-rte/src/core/BlockElement.tsx

```diff
@@ -12,81 +12,108 @@ type StylableBlockTypes = Extract<
 
 export type RteBlockElementClassKey = StylableBlockTypes | "root";
 
-export interface RteBlockElementProps extends TypographyProps {
+type OwnerState = Pick<RteBlockElementProps, "type">;
+
+const Root = styled(Typography, {
+    name: "CometAdminRteBlockElement",
+    slot: "root",
+    overridesResolver({ ownerState }: { ownerState: OwnerState }, styles) {
+        return styles.root, ownerState.type === "header-one";
```

Comment:
This should include the style of the given `type`

```suggestion
        return styles.root, styles[ownerState.type];
```